templates/quiz-result-overview.html

```diff
@@ -0,0 +1,121 @@
+                            {% if result_items[i].index_chosen == 0 %}
+                                {{ triangle|safe }}
+                            {% elif result_items[i].index_chosen == 1 %}
+                                {{ diamond|safe }}
+                            {% elif result_items[i].index_chosen == 2 %}
+                                {{ square|safe }}
+                            {% elif result_items[i].index_chosen == 3 %}
+                                {{ circle|safe }}
+                            {% elif result_items[i].index_chosen == 4 %}
+                                {{ pentagram |safe }}
+                            {% elif result_items[i].index_chosen == 5 %}
+                                {{ triangle_6|safe }}
+                            {% endif %}
+                        </p>
+                    {% endif %}
+
+                </div>
+                <div class="grid-item grid grid-cols-2 gap-x-4">
+                    <div class="table-el bg-blue-200 shadow-lg flex flex-col-75 place-content-center rounded-b-md">
+
+                        {% if "option_text" in result_items[i] %}
+                            <p class="flex pl-6  text-center font-slab text-2xl question-text text-2xl"> {{ result_items[i].option_text | nl2br }}</p>
+                        {% elif "code" in result_items[i] %}
+                            <pre><code
+                                    class="pl-6 flex block text-l">{{ result_items[i].code | nl2br }}</code></pre>
+                        {% endif %}
+                    </div>
+
+                    {% if result_items[i].is_correct and result_items[i].attempts ==  1 %}
+                        <p class="place-content-center table-el text-center bg-green-200 text-xl font-slab flex m-0 rounded-b-md flex-col-25 mr-4">
+                            In {{ result_items[i].attempts - 1 }} poging </p>
+                    {% elif result_items[i].is_correct and result_items[i].attempts > 1 %}
+                        <p class="place-content-center table-el text-center bg-green-200 text-xl font-slab flex m-0 rounded-b-md flex-col-25 mr-4">
+                            In
+                            {{ result_items[i].attempts - 1 }}
+                            pogingen </p>
+                    {% else %}
+                        <p class="place-content-center table-el text-center bg-red-200 text-xl font-slab flex m-0 rounded-b-md flex-col-25 mr-4">
+                            In
+                            {{ result_items[i].attempts - 1 }}
+                            pogingen </p>
+                    {% endif %}
+                </div>
+
+            </div>
+        {% endfor %}
+    </div>
+    <div class="p-10 button-bar border-t-8 border-green-600">
+        <div></div>
+        <button class="green-btn ltr:ml-1 rtl:mr-1"
+                onclick="window.parent.location = '{{ hedy_link(level, next_assignment) }}';"> {{ ui.go_to_level }} {{ level }} </button>
```

Comment:
In this case we do use the `hedy_link` function. Would be nice to do this consistent, either everywhere on not at all.